hack/generator/pkg/astmodel/package_reference_test.go

```diff
@@ -6,14 +6,13 @@
 package astmodel
 
 import (
-	"github.com/onsi/gomega"
 	"testing"
+
+	"github.com/onsi/gomega"
```

Comment:
Isn't this supposed to be brought into scope?
```
. "github.com/onsi/gomega"
```

Then you can write
```
g := NewGomegaWithT(t)
```

Reply:
changed